insights/parsers/nvidia.py

```diff
@@ -3,14 +3,28 @@
 ==========================
 
 NvidiaSmiL - command ``/usr/bin/nvidia-smi -L``
-------------------------------------------------
+NvidiaSmiActiveClocksEventReasons - command ``/usr/bin/nvidia-smi --query-gpu=name,clocks_event_reasons.active --format=csv,noheader``
+--------------------------------------------------------------------------------------------------------------------------------------
 """
```

Comment:
For multiple parsers entries case, this docstings `---` usage should be something like: 
```
DiskFree_LI - command ``df -li``
--------------------------------

DiskFree_ALP - command ``df -alP``
----------------------------------
```